addon/components/list-card.js

```diff
@@ -248,3 +287,6 @@ export default Ember.Component.extend({
     }
   }
 });
+
+export default ListCardComponent;
+export { ListCardComponent, QueryToken, QueryFilter, FilterGroup };
```

Comment:
ya, I actually wanted to bring this up but forgot....I think it would be much nicer to have them separate and since they are Ember.Object's....maybe put them in `addon/classes/` or something